common4j/src/main/com/microsoft/identity/common/java/nativeauth/providers/interactors/SignInInteractor.kt

```diff
@@ -61,12 +62,20 @@ class SignInInteractor(
     ): SignInInitiateApiResult {
         LogSession.logMethodCall(
             tag = TAG,
-            correlationId = parameters.correlationId,
+            correlationId = parameters.getCorrelationId(),
             methodName = "${TAG}.performSignInInitiate(parameters: SignInStartCommandParameters)"
         )
+
         val request = nativeAuthRequestProvider.createSignInInitiateRequest(
             commandParameters = parameters
         )
+
+        Logger.verbose(
+            "${TAG}.performSignInInitiate",
+            parameters.getCorrelationId(),
+            "performSignInInitiate: request = $request"
```

Comment:
Should we be logging the full requests + responses? Could they contain potentially sensitive information like sent passwords, codes, etc?

Reply:
Excellent point Robert, and coincidentally: https://identitydivision.visualstudio.com/Engineering/_boards/board/t/DevExDub%20-B2C/Backlog%20items/?workitem=2900824

@Yuki-YuXin let's hold off with this PR for now. I need to work together with you to ensure we're being even more careful with logging.